README.md

```diff
@@ -18,7 +18,11 @@ NOTE: This has a peer dependencies of `rxjs@5.0.*` and `redux`, which will have
 as well.
 
 ```sh
-npm install --save redux-observable
```

Comment:
If instead you want to clarify the process better, the full instructions could be amended to the `examples/basic/README.md` to be:

```
git clone https://github.com/redux-observable/redux-observable.git
cd redux-observable
npm install
cd examples/basic/
npm install
npm start
```